data_steward/cdr_cleaner/cleaning_rules/remove_aian_participants.py

```diff
@@ -0,0 +1,71 @@
+"""
+Background
```

Comment:
I like the module doc string.  Thanks for adding it!  Just a PEP-8 headsup.  All documentation strings (module, class, method, or function) should start with a single line summary followed by a blank line.  This will allow tool generated documentation in the future.